crates/atuin-daemon/src/client.rs

```diff
@@ -15,12 +19,22 @@ pub struct HistoryClient {
 
 // Wrap the grpc client
 impl HistoryClient {
-    pub async fn new(path: String) -> Result<Self> {
+    // Allow unused because the port won't be used on unix and the path won't be on non-unix.
```

Comment:
Instead, we could define two separate `new` functions. One for windows and one for unix. Then we don't need the unused var